### PR TITLE
Bugfix/fix node count config name

### PR DESCRIPTION
**Pull request by talekhinezh**

### CODE REVIEW

File: radixdlt/src/integration/java/com/radixdlt/consensus/CrashFaultNetworkTest.java [hunk 1]
```diff
@@ -20,6 +20,7 @@
 import com.radixdlt.crypto.ECKeyPair;
 import com.radixdlt.crypto.ECPublicKey;
 import com.radixdlt.examples.tictactoe.Pair;
```
Comment (clstrfsck) [inline radixdlt/src/integration/java/com/radixdlt/consensus/CrashFaultNetworkTest.java]: Oh, that's not great (import from examples).  I didn't notice that before.

**Review by flotothemoon — APPROVED** [review]

File: radixdlt/src/main/java/org/radix/Radix.java [hunk 2]
```diff
@@ -71,6 +77,22 @@
 	private static final Object BC_LOCK = new Object();
 	private static boolean bcInitialised;
 
+	// Be careful here.  This is a debugging method to detect overcommitted heap usage on Linux.
```
Comment (clstrfsck) [inline radixdlt/src/main/java/org/radix/Radix.java]: This should not be here.  Need to rebase on current head of `feature/rpnv1-67-oom-killer-linux`

**Review by clstrfsck — CHANGES_REQUESTED** [review]

File: radixdlt/src/integration/java/com/radixdlt/consensus/CrashFaultNetworkTest.java (hunk 1 above)
Comment (talekhinezh) [reply radixdlt/src/integration/java/com/radixdlt/consensus/CrashFaultNetworkTest.java]: confused why this file is even showing up here

**Review by clstrfsck — APPROVED** [review]
Looks good when we resolve the conflict.

**Pull request merged**